packages/babel-cli/test/fixtures/babel-node/v8Flag-dashed-with-param/options.json

```diff
@@ -1,4 +0,0 @@
-{
-  "args": ["--expose-debug-as=customDebug", "--eval", "console.log(customDebug.Debug.DebugEvent.Break)"],
-  "stdout": "1"
-}
```

Comment:
Instead of removing can we maybe use `--expose-gc-as` (with dashes) here?

the tests triy to confirm that we support all v8 flags either dashed (`-`) or with underscore (`_`)